lib/ssl/doc/src/ssl.xml

```diff
@@ -1748,11 +1748,15 @@ fun(srp, Username :: binary(), UserState :: term()) ->
     
     <func>
       <name name="eccs" arity="0" since="OTP 19.2"/>
+      <fsummary>Returns a list of all supported elliptic curves.</fsummary>
+      <desc><p>Returns a list of all supported elliptic curves pre TLS-1.3.</p>
+      </desc>
+    </func>
+
+    <func>
       <name name="eccs" arity="1" since="OTP 19.2"/>
-      <fsummary>Returns a list of supported ECCs.</fsummary>
-      <desc><p>Returns a list of supported ECCs. <c>eccs()</c>
-      is equivalent to calling <c>eccs(Protocol)</c> with all
-      supported protocols and then deduplicating the output.</p>
+      <fsummary>Returns the by default supported elliptic curves.</fsummary>
```

Comment:
some word is missing in the sentence?
AFAIK fsummary is description being displayed in IDEs ... then the description should clarify when arity 1 function should be used instead of arity 0.

Reply:
Made it same as description